Organic/ConnatixConfig.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Organic;
+
+class ConnatixConfig extends BaseConfig {
+    const DEFAULT_AMP_RELATIVE = 'after';
+    const DEFAULT_AMP_SELECTORS = [ 'p:first-child', 'span:first-child' ];
+    const DEFAULT_CONFIG = [
+        'enabled' => false,
+        'playspaceId' => '',
+    ];
+
+    /**
+     * @var Organic
+     */
+    private $organic;
+
+    /**
+     * @var bool
+     */
+    private $enabled;
+
+    /**
+     * @var string
+     */
+    private $playspaceId;
+
+    public function __construct( Organic $organic ) {
+
+        $rawAdsConfig = $organic->getOption( 'organic::ad_settings', [] );
+        $raw = isset( $rawAdsConfig['adSettings']['connatix'] )
```

Comment:
As I mentioned [in the review for other PR](https://github.com/orgnc/platform/pull/2005#pullrequestreview-1063106753) we shouldn't use SDK-level config for Connatix.

Instead, I think we should:
1) Use Placement-level setting by default (as you are already doing)
2) Use Plugin-level setting only for backward compatibility if there was no Placement for Connatix in the Ad config

Reply:
Ok, makes sense, i'll close platform PR.